content/en/docs/tasks/extend-kubernetes/custom-resources/custom-resource-definitions.md

```diff
@@ -761,6 +761,85 @@ Default values must be pruned (with the exception of defaults for `metadata` fie
 
 Default values for `metadata` fields of `x-kubernetes-embedded-resources: true` nodes (or parts of a default value covering `metadata`) are not pruned during CustomResourceDefinition creation, but through the pruning step during handling of requests.
 
+#### Nullable Defaults
+
+If a field is non-nullable as defined in the OpenAPI schema and a default is specified, null values will assume the value of the default. There is a distinction between null values and omitted values. For example, if the CustomResourceDefinition schema looks like
+
+```yaml
+type: object
+properties:
+  spec:
+    type: object
+    properties:
+      foo:
+        type: string
+        nullable: false
+        default: "default"
+      bar:
```

Comment:
Maybe also give this a default

Reply:
I'm assuming the goal is to see the difference between nullable and non-nullable.